Experimental/BenchmarkHashes/psykal_lite_mod.F90

```diff
@@ -0,0 +1,2669 @@
+!-----------------------------------------------------------------------------
```

Comment:
I know this comes from another project but I think we should modify this copyright header in some way, or add a clarifying line at the end to make it clear that this is not applicable to the Fab project and is just from this example file's original implementation

Reply:
And done.
I've copied the licence text from the donor project, as I should have done in the first place.